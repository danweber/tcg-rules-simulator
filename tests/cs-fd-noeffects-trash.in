VERSION:2

# the facedown inherited shouldn't activate in trash

FIELD1:Inherited,DW1-Van,DW1-Rusher,FACEDOWN,DW1-Rusher,FACEDOWN,DW1-Rusher
#FACEDOWN,CS1-Paper,CS1-Paper,FACEDOWN,CS2-01 Red,CS2-ShieldSmasher
GAME:P1 T3 MAIN 5
SECURITY2:DW1-Big

DUMP

CMD:ATTACK Inherited Player
STEP:AUTOANSWER:CAPTURE
ECHO: -- gain only 1 memory
DUMP



