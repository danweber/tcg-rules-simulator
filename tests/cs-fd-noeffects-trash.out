FIELD1:Inherited,DW1-Vanillamon,DW1-Rusher,FACEDOWN,DW1-Rusher,FACEDOWN,DW1-Rusher
FIELD2:
GAME:P1 T3 MAIN 5
ECHO: -- gain only 1 memory
FIELD1:
FIELD2:
GAME:P1 T3 MAIN 6
